util/keyfile/src/lib.rs

```diff
@@ -59,7 +61,50 @@ pub fn read_pubfile_data<R: std::io::Read>(
     Ok(result)
 }
 
-fn to_io_error(err: serde_json::error::Error) -> std::io::Error {
+/// Write user b58 public address to disk
+pub fn write_b58pubfile<P: AsRef<Path>>(
+    path: P,
+    addr: &PublicAddress,
+) -> Result<(), std::io::Error> {
+    let mut wrapper = PrintableWrapper::new();
+    wrapper.set_public_address(addr.into());
+
+    let data = wrapper.write_to_bytes().map_err(to_io_error)?;
+    let data = bs58::encode(data).into_vec();
+
+    File::create(path)?.write_all(&data)?;
+    Ok(())
+}
+
+/// Read user b58 public address from disk
+pub fn read_b58pubfile<P: AsRef<Path>>(path: P) -> Result<PublicAddress, std::io::Error> {
+    read_b58pubfile_data(&mut File::open(path)?)
+}
+
+/// Read user b58 pubfile from any implementor of `Read`
+pub fn read_b58pubfile_data<R: std::io::Read>(
+    buffer: &mut R,
+) -> Result<PublicAddress, std::io::Error> {
+    let data = {
+        let mut data = Vec::new();
+        buffer.read_to_end(&mut data)?;
+        data
+    };
+    let data = bs58::decode(data).into_vec().map_err(to_io_error)?;
```

Comment:
Should use https://github.com/mobilecoinfoundation/mobilecoin/blob/master/api/src/display.rs#L54